kentik_api_library/kentik_api/api_resources/alerting_api.py

```diff
@@ -11,3 +14,57 @@ def create_manual_mitigation(self, manual_mitigation: ManualMitigation) -> bool:
         api_call = alerts.create_manual_mitigation()
         response = self._send(api_call, manual_mitigation)
         return manual_mitigations_payload.CreateResponse.from_json(response.text).status() == "OK"
+
+    def get_active_alerts(
+        self,
+        start_time: datetime,
+        end_time: datetime,
+        filter_by: Optional[str] = None,
+        filter_val: Optional[str] = None,
+        show_mitigations: int = 1,
+        show_alarms: int = 1,
```

Comment:
these ints that take only 0 or 1 actually represent bool flags, please use bool instead